recipe/meta.yaml

```diff
@@ -107,7 +110,7 @@ outputs:
 
 about:
   home: https://curl.haxx.se/
-  license: curl
+  license: MIT
```

Comment:
Can you please verify this? As far as I know, curl uses an MIT-like license but it isn't actually an MIT license.

Reply:
It's a curl license https://curl.se/docs/copyright.html. We can verify it here too https://spdx.org/licenses/

Reply:
@skupr-anaconda So line 110 needs to be switched back to `curl`?

Reply:
I went ahead and changed it back to `curl`